CHANGELOG.md

```diff
@@ -39,6 +40,17 @@
 - [v0.2.0](#v020)
 - [v0.1.0](#v010)
 
+## [v1.13.0]
```

Comment:
What if there is another patch release?
Also, users may not actually read  'Release date: TBD' and get confused.
Please rename this header to 'Unreleased'.

Reply:
Roger that, makes sense. Fixed!